hardware/Honeywell.cpp

```diff
@@ -314,7 +361,7 @@ void CHoneywell::SendSetPointSensor(const unsigned char Idx, const float Temp, c
 	thermos.id2 = 0;
 	thermos.id3 = 0;
 	thermos.id4 = Idx;
-	thermos.dunit = 0;
+	thermos.dunit = 0; 
```

Comment:
Please undo whitespace at the end of the line

Reply:
done